### ISSUE TITLE
whenever i try to download litematica, the file keeps becoming an audio or video of some sort

### ISSUE THREAD

**Issue by aaaaaaaaaaaaaaaaaaaaaaaaaas** [maruohon/litematica#526]

**Comment by Felix14-v2** [comment]
Please, attach more additional context like screenshots or screen record.

**Comment by aaaaaaaaaaaaaaaaaaaaaaaaaas** [comment]
[image: Screenshot 2022-04-30 133421.png] I try to open it and this shows
[image: Screenshot 2022-04-30 133409.png] I completed the download
[image: Screenshot 2022-04-30 133359.png] I save it
[image: Screenshot 2022-04-30 133347.png] It starts to download it
[image: Screenshot 2022-04-30 133335.png] I click download

**Comment by maruohon** [comment]
You know none of those images actually show up here...

**Comment by aaaaaaaaaaaaaaaaaaaaaaaaaas** [comment]
interesting, let me try again i guess

**Comment by aaaaaaaaaaaaaaaaaaaaaaaaaas** [comment]
[image: image.png]

does it work?

**Comment by maruohon** [comment]
No, I don't think it will work via some email replies

**Comment by aaaaaaaaaaaaaaaaaaaaaaaaaas** [comment]
I made this an attachment this time, if it doesn't work I'll try to screen
record
